### PR TITLE
TPC: added QC plots for DCS points - Temperature

### PR DESCRIPTION
**Pull request by tklemenz**

### CODE REVIEW

**Comment by tklemenz** [comment]
This PR needs https://github.com/AliceO2Group/AliceO2/pull/8745 in O2.

File: Modules/TPC/src/DCSPTemperature.cxx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      getObjectsManager()->addMetadata(canv->GetName(), key, value);
+    }
+  }
+}
+
+void DCSPTemperature::update(Trigger t, framework::ServiceRegistry&)
+{
+  std::vector<long> usedTimestamps = getDataTimestamps("TPC/Calib/Temperature", mNFiles, mTimestamp);
+  for (auto& timestamp : usedTimestamps) {
+    mData.emplace_back(mCdbApi.retrieveFromTFileAny<o2::tpc::dcs::Temperature>("TPC/Calib/Temperature", mLookupMap, timestamp));
```
[the remark this answers is not in the document]
Comment (tklemenz) [reply Modules/TPC/src/DCSPTemperature.cxx:100]: I made the temperature Objects unique pointers. Should also work, right?

**Review by wiechula — APPROVED** [review]

File: Modules/TPC/src/DCSPTemperature.cxx (hunk 1 above)
[the remark this answers is not in the document]
Comment (tklemenz) [reply Modules/TPC/src/DCSPTemperature.cxx:99]: It compiles and it is running. The output of the task is what I expect as far as I can test it. Is there a way to make sure that everything is deleted properly?

[the remark this answers is not in the document]
Comment (wiechula) [reply Modules/TPC/src/DCSPTemperature.cxx:99]: if you can run without crash it's fine. I trust `unique_ptr` does the correct thing.

**Comment by wiechula** [comment]
@Barthelemy , @knopers8 , the macOS failure looks unrelated, can this be merged?

**Pull request merged**